### DISCUSSION TITLE
Language change

### DISCUSSION THREAD

**Discussion by ghost** [valnoxy/t480-oc#66] [General]
Is there a work around or german menu to use in german  explanation in english ?
en-US:0 , en_US:0 wont work. still german language and i wont able to change to drop down menu
Edit: keyboard changes but not the language.
Edit2: Loops on instalatio - never completes and no wifi detected.

**Comment by valnoxy** [comment]
In this case you have to reset your NVRAM, but be careful! Do not use the tool provided by OpenCore, because it will brick your mainboard. It is best to perform a BIOS update to the latest version (in-place upgrade is also possible). 

Can you tell me more about your hardware?
What CPU and WiFi card do you have? It is often the case that WiFi is not possible in the setup. If possible, try the installation via Ethernet.
